packages/trpc/server/routers/viewer/teams/create.handler.ts

```diff
@@ -60,6 +71,9 @@ export const createHandler = async ({ ctx, input }: CreateOptions) => {
         requestedSlug: slug,
       },
       ...(!IS_TEAM_BILLING_ENABLED && { slug }),
+      parent: {
+        connect: parentId ? { id: parentId } : undefined,
+      },
```

Comment:
If we do have an org id present - we connect this team to have a parent that is the org